.github/workflows/dns-apply.yml

```diff
@@ -23,6 +23,7 @@ jobs:
       - uses: cachix/install-nix-action@d1ca217b388ee87b2507a9a93bf01368bde7cec2 # v31
       - name: dnscontrol push
         env:
+          GANDI_TOKEN: "${{ secrets.GANDI_TOKEN }}" # Expires 2026-04-07
           NETLIFY_TOKEN: "${{ secrets.NETLIFY_TOKEN }}"
```

Comment:
(I'm assuming it'll work find to deploy to netlify and gandi in parallel). Suggestion: consider adding a TODO/issue to remind us to clean up netlify (not just this code, but the old dns entries in netlify to avoid confusion).